website/docs/embedded_components.md

```diff
@@ -1,18 +1,20 @@
 ---
 id: embedded_components
-title: Embedded components
+title: Embedded components/JBrowse Jupyter/JBrowseR
```

Comment:
I don't like the look of the /'s, I think

Embedded components, JBrowse Jupyter, and JBrowse R

would look better

Reply:
Changed to commas :) keeping the JBrowseR notation though